tests/test_other.py

```diff
@@ -11996,6 +11996,7 @@ def test_multiple_g_flags(self):
     self.run_process([EMCC, test_file('hello_world.c'), '-c', '-g', '-g0'])
     self.assertNotIn(b'.debug', read_binary('hello_world.o'))
 
+  @require_v8
```

Comment:
You also don't need `engine=v8` below anymore.

And you can replace `v8 = ` with `self.v8_args += [`